scripts/server/main.js

```diff
@@ -96,9 +97,9 @@ fastify.register(fastifyMultipart, {
   attachFieldsToBody: true,
   limits: {
     fieldNameSize: 100, // Max field name size in bytes
-    fieldSize: 100,     // Max field value size in bytes
+    fieldSize: 1000,     // Max field value size in bytes
     fields: 10,         // Max number of non-file fields
-    fileSize: 1000000,  // For multipart forms, the max file size in bytes
+    fileSize: 200000000,  // For multipart forms, the max file size in bytes
```

Comment:
```suggestion
    fileSize: 200_000_000,  // For multipart forms, the max file size in bytes
```

Mes yeux ont du mal à lire avec le nombre de zeros :-p

J'utilise ça : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#numeric_separators

Reply:
Et maintenant que j'arrive à mieux lire, j'ai l'impression que 200Mo, ça fait beaucoup ?
Est-ce que tu voulais dire 20Mo ?
ou sinon, est-ce que tu peux expliquer ton raisonnement derrière cette valeur ?